bin/cml/comment.js

```diff
@@ -0,0 +1,8 @@
+exports.command = 'comment';
```

Comment:
`cml comment --help` doesn't show options like `cml pr --help` does, both are functionally the same with respect to the `create` subcommand?

Reply:
Thanks to “Obsessive Backwards Compatibility”, some “nouns” like `cml pr` and `cml runner` leak the “verb” options to the parent command. Won't fix without breaking changes. 🙉

Reply:
so what is the correct behavior? to show the "default" subcommand's options or not?

Reply:
To **not** show any of the “verb” (child) command in the “noun” (parent) command's help menu. Options should always propagate from parent to child commands (downwards), not upwards.

Reply:
Resolved with https://github.com/iterative/cml/pull/1073/commits/336a3af1d49a8e6e4d1392498fe90589e9691c3b